bazel/test.bzl

```diff
@@ -161,6 +162,7 @@ def redpanda_cc_gtest(
         custom_args = args,
         env = env,
         data = data,
+        extra_copts = ["-DIS_GTEST"],
```

Comment:
It is more idiomatic to use defines or local defines: https://bazel.build/reference/be/c-cpp#cc_binary.local_defines

Reply:
Done